src/include/migraphx/instruction.hpp

```diff
@@ -37,10 +37,17 @@ struct instruction
     bool valid() const;
 
     shape get_shape() const;
+
+    void set_shape(shape);
```

Comment:
Remove this, use `recompute_shape` instead.

Reply:
see reply to "replace_argument" below.  Same issue.